internal/rollup/rollup.config.js

```diff
@@ -193,6 +194,7 @@ const config = {
     banner,
     format: 'TMPL_output_format',
   },
+  preserveSymlinks: true,
```

Comment:
Nice. Do these preserve symlinks changes work as intended to keep the resolve within the cwd()?

Reply:
Yep, in this particular case it was really the `jail` attribute that brought the non-hermetic build to fail.